plugins/lando-services/blackfire/blackfire.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Lando blackfire service builder
+ *
+ * @name blackfire
+ */
+
+'use strict';
+
+module.exports = function(lando) {
+
+  // Modules
+  var _ = lando.node._;
+
+  /**
+   * Supported versions for blackfire
+   */
+  var versions = [
+    '1.15.0',
+    'latest',
+    'custom'
+  ];
+
+  /**
+   * Return the networks needed
+   */
+  var networks = function() {
+    return {};
+  };
+
+  /**
+   * Build out blackfire
+   */
+  var services = function(name, config) {
+
+    // Start a services collector
+    var services = {};
+
+    // Default blackfire service
+    services[name] = {
+      image: 'blackfire/blackfire:' + config.version,
+      environment: {
+        BLACKFIRE_SERVER_ID: process.env.BLACKFIRE_SERVER_ID,
+        BLACKFIRE_SERVER_TOKEN: process.env.BLACKFIRE_SERVER_TOKEN
+      },
+      command: 'blackfire-agent'
```

Comment:
Shouldn't the command be 'docker-entrypoint.sh blackfire-agent'. This is how SOLR, Mongodb and such setup the command.

Reply:
these look on the host machine; let's take these out for now in favor of setting them in the `.env` file.

Reply:
@jbertoen, `docker-entrypoint.sh` is a wrapper provided by the solr and mongodb containers (see https://github.com/docker-solr/docker-solr/blob/9df068583b90e82844d2bf6cfe481d23c2840636/7.2/Dockerfile#L65).  It's not necessary here.

@serundeputy's right about the variables... all host environment variables (including those in .env) are passed into all containers, so there's no need to define them manually.